Proxy/proxy.sample.ini

```diff
@@ -6,9 +6,16 @@ PORT=8000
 FILENAME=log.db
 SCHEMANAME=db.sql
 
+[TESTDATABASE]
+FILENAME=log.test.db
+
 [PROXY]
 LOG=0
 UNITS=1
```

Comment:
@jsr38 Yeah this is an area where it would help if @kb1lqd and I documented it a bit better. We do need to spend some time on this type of documentation. The port number is essentially an eight bit number (0-255) that indicates a potential application on Faraday itself written in firmware. For example `port 5` is the telemetry application on Faraday so any data received from port 5 will be related to telemetry. Any 'port 2' data is intended for the command/control application that controls GPIO and other settings. Think of it as the UART COM is an "IP Address" and the `Port` is the one of many ports available on an IP address (i.e. 80 for HTTP, 22 for SSH, etc)... Similar concept